js/src/dapps/registry/Lookup/lookup.js

```diff
@@ -86,14 +102,26 @@ export default class Lookup extends Component {
     );
   }
 
-  onNameChange = (e) => {
-    this.setState({ name: e.target.value });
+  onInputChange = (e) => {
+    this.setState({ input: e.target.value });
   };
   onTypeChange = (e, i, type) => {
     this.setState({ type });
-    this.props.actions.clear();
+    this.props.clear();
```

Comment:
Spacing between functions as per the preceding comment. (While we are cleaning up the code here, take the opportunity to make it a bit more maintainable)